Firmware/PlatformIO/main_mcu/src/bms.cpp

```diff
@@ -19,6 +27,31 @@ void BMS::Tick(std::chrono::milliseconds elapsed_time)
     // todo
 }
 
+// Find maximum discharge and regen current
+void BMS::GetStateOfEnergy()
+{
+    // Update min voltage (max already set(?))
+    minVoltage = *std::min_element(voltages.begin(), voltages.end());
```

Comment:
This should probably be done in ProcessState since multiple functions will use it